src/plugins/three/LoadRegionPlugin.d.ts

```diff
@@ -4,8 +4,9 @@ import { TilesRenderer } from '../../three/TilesRenderer.js';
 import { OBB } from '../../three/math/OBB.js';
 import { Sphere, Ray } from 'three';
 
-declare class Region {
+declare class BaseRegion {
```

Comment:
This should probably be declared as an "interface" rather than a class. It's okay to pass an object into the region functions:

```js
plugin.addRegion( {
  errorTarget: 10,
  intersectsTile: /* ... */
} )
```

Reply:
Interfaces in TypeScript cannot have constructors. I could create both: `export class BaseRegion implements IRegion` and have `addRegion(region: IRegion): void;`?

Reply:
A constructor shouldn't be important for the object being passed to the class and better captures the original design intent of the function, I think. But these kinds typing decisions are not really where I enjoy spending my time, unfortunately, so I'll have to lean on those who are more familiar with using them 😅